File: School-Management-System/WebApi/ClientApp/src/app/home/components/certificate/model/studentCertificate.ViewModel.ts
export interface StudentCertificateViewModel {
  id: string;
  academicYear: string;
  admittedDate: Date;
  issueDate: Date;
  name: string;
  age: number;
  gender: number;
  fatherName: string;
  motherName: string;
  dateOfBirthNp: string;
  dateOfBirthEn: string;
  address: string;
  registrationNumber?: string | null;
  symbolNumber?: string | null;
  wardNo: number;
  classRoom: string;
  firstAdmittedClass: string;
  section: string;
  gpa: number;
  examType: string;
  examHeld: Date;
  isCharacterCertificateTaken:boolean;
  isTransferCertificateTaken:boolean;
}

File: School-Management-System/Application/Students/ViewModels/StudentCertificateViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.Students.ViewModels
{
    public class StudentCertificateViewModel
    {
        public Guid Id { get; set; }
        public Guid StudentId { get; set; }
        public string AcademicYear { get; set; }
        public DateTime AdmittedDate { get; set; }
        public DateTime IssueDate { get; set; }
        public string Name { get; set; }
        public int? Age { get; set; }
        public int Gender { get; set; }
        public string FatherName { get; set; }
        public string MotherName { get; set; }
        public string DateOfBirth { get; set; }
        public string Address { get; set; }
        public string? RegistrationNumber { get; set; }
        public string? SymbolNumber { get; set; }
        public int WardNo { get; set; }
        public string ClassRoom { get; set; }
        public string FirstAdmittedClass { get; set; }
        public string Section { get; set; }
        public decimal GPA { get; set; }
        public int ExamType { get; set; }
        public DateTime ExamHeld { get; set; }
        public bool IsCharacterCertificateTaken { get; set; }
        public bool IsTransferCertificateTaken { get; set; }
    }
}
